src/pycall/julia_helpers.jl

```diff
@@ -1,6 +1,6 @@
 import Pkg
 
-if !haskey(Pkg.installed(), "PyCall") # FIXME: Pkg.installed() is deprecated!
```

Comment:
there's a proper way to do this via https://pkgdocs.julialang.org/v1/creating-packages/#Conditional-loading-of-code-in-packages-(Extensions)

but I can do that PR when you figure out features

Reply:
If you just need a drop-in replacement for `Pkg.installed()` you can use this:

```julia
isinstalled(pkg::String) = any(x -> x.name == pkg && x.is_direct_dep, values(Pkg.dependencies()))
```